### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by razzeee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,4 +160,4 @@ It is possible to refactor an existing SPA into this style.  Add this kind of wr
 
 If a Page needs to be able to invoke global operations, it can store a request in the `Session` that `Main.elm` could look at after calling the page's `update`.  There's no example of that in here (yet), though. )
 
-Bug:  If you try to reach an authenticated page without auth, you'll get sent to Login, and thereafter redirected to the page.  The current router leaves the URL at '/login', though.  It is on my queue to work this.
\ No newline at end of file
+Bug:  If you try to reach an authenticated page without auth, you'll get sent to Login, and thereafter redirected to the page.  The current router leaves the URL at '/login', though.  It is on my queue to work this.
```
Comment (jhbrown94) [inline README.md]:
```suggestion
Bug:  If you try to reach an authenticated page without auth, you'll get sent to `Page.Login`, and thereafter redirected to the page.  The current router leaves the URL at '/login', though.  It is on my queue to work this.
```

**Review by jhbrown94 — CHANGES_REQUESTED** [review]
Thanks for the attention to detail!  I've proposed a different edit (to a change that was perfectly reasonable) in one case.

**Comment by razzeee** [comment]
Added

**Review by jhbrown94 — APPROVED** [review]
Thank you!

**Comment by jhbrown94** [comment]
Typo fixes from @Razzeee.

**Comment by jhbrown94** [comment]
Whoops, I approved without merging.  @Razzeee  it looks like you blew away the branch already -- can you restore it?  If not, I can manually apply those changes directly on master easily enough and credit you in the commit message.

**Comment by razzeee** [comment]
Yeah, saw that too late. Branch is gone for good. Just fix it no credit needed :)

**Pull request closed without merging**